src/Autocomplete/assets/src/controller.ts

```diff
@@ -142,20 +142,21 @@ export default class extends Controller {
             // VERY IMPORTANT: use 'function (query, callback) { ... }' instead of the
             // '(query, callback) => { ... }' syntax because, otherwise,
             // the 'this.XXX' calls inside this method fail
-            load: function (query: string, callback: (results?: any) => void) {
+            load: function (query: string, callback: (options?: any, optgroups?: any) => void) {
```

Comment:
After a little digging, it looks like this callback as a type:

```suggestion
            load: function (query: string, callback: TomLoadCallback) {
```

The `TomLoadCallback` can be imported from `'tom-select/dist/types/types'` like 3 other types already. Then, for the `.catch()` on line 154, it would become:

```
.catch(() => callback([], []));
```

to satisfy this type.